src/containers/User/Login.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router'
+import { Link } from 'react-router-dom'
+import { Form, Button, Container, Grid, Segment, Header } from 'semantic-ui-react'
+import config from '../../config.json'
+import isLoggedIn from '../../utils/helpers/loginCheck'
+
+const Login: React.FC = () => {
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+  const [isError, setIsError] = useState(false)
+  const history = useHistory()
+
+  const handleSubmit = async (event: any) => {
+    event.preventDefault()
+    const passwordHash = hashPassword(password)
+    const loginResult = await attemptLogin(username, passwordHash)
+    if (!loginResult.success) setIsError(true)
+    else {
+      setIsError(false)
+      // localStorage.setItem('username', username)
+      // localStorage.setItem('templatePermissions', loginResult.templatePermissions)
+      localStorage.setItem('persistJWT', loginResult.JWT)
+      console.log('Log in successful!')
```

Comment:
good time to set username and templatePermissions

https://github.com/openmsupply/application-manager-web-app/blob/0da5d6491908ef9676537689e8be4319892d8ad1/src/utils/hooks/useGetUserPermissions.ts#L21-L22